Source/Internal/Table/PDFTableObject.swift

```diff
@@ -450,7 +461,22 @@ internal class PDFTableObject: PDFRenderObject {
         }
         return result
     }
-
+    
+    internal typealias ShiftedCells = ([PDFTableCalculatedCell])
```

Comment:
```suggestion
    internal typealias ShiftedCells = [PDFTableCalculatedCell]
```
It is not necessary to wrap the array in parantheses and that might lead to side effects (it would be a single value tuple)